src/containers/ThankYouMsgContainer.js

```diff
@@ -0,0 +1,17 @@
+import { connect } from 'react-redux';
+import ThankYouMsg from '../components/ThankYouMsg';
+
+const mapStateToProps = (state) => {
+  return {};
+};
+
+const mapDispatchToProps = (dispatch) => {
+  return {};
+};
+
+const ThankYouMsgContainer = connect(
+  mapStateToProps,
+  mapDispatchToProps
+)(ThankYouMsg);
+
+export default ThankYouMsgContainer;
```

Comment:
This container isn't doing anything. Unless it is going to be fleshed out down the road, this should be removed.